resources/spec.yaml

```diff
@@ -14671,6 +14684,31 @@ definitions:
     x-okta-parent: '#/definitions/ApplicationSettingsApplication'
     x-okta-tags:
       - Application
+  RadiusApplication:
```

Comment:
Can we add new ITs / update existing ITs to use these new models?

Reply:
https://github.com/okta/okta-sdk-golang/pull/266